test/IniModuleLoader.test.cpp

```diff
@@ -0,0 +1,13 @@
+#include "IniModuleLoader.h"
+#include <gtest/gtest.h>
+
+TEST(IniModuleLoader, InitializeNoModules)
+{
+	IniModuleLoader iniModuleLoader;
+	EXPECT_NO_THROW(iniModuleLoader.LoadModules());
+	
+	EXPECT_EQ("", iniModuleLoader.GetModuleName(0));
+	EXPECT_EQ("", iniModuleLoader.GetModuleName(1));
```

Comment:
My previous suggestion would affect this test, so it would instead be expected to throw.